Test/civl/regression-tests/non-interference-2.bpl.expect

```diff
@@ -2,10 +2,8 @@ non-interference-2.bpl(25,1): Error: Non-interference check failed
 Execution trace:
     non-interference-2.bpl(7,3): anon0
     non-interference-2.bpl(12,5): inline$AtomicIncr$0$anon0
-    (0,0): inline$Civl_NoninterferenceChecker_yield_PC$0$L
```

Comment:
I get that (0, 0) is not useful, but what about the label on the right? It looked useful.

Reply:
Yeah.. that name is still in the codebase, but someone needs to assign a proper token to some of the generated nodes before it will show up in the execution trace. I can't really judge what the civl code is doing and what tokens should be where.

Reply:
Where is the message stored? If it's not in the token, we should display it without token information. If it was in the token, and is lost because of the new encoding, I'd recommend to revert that change.
I worry about breaking Civi users who relied on the last line being printed.

Reply:
Okay, I've made it so that nothing changes in the civl tests.